<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polygon Illustrator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="my_label" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>221</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Fill Algorithm:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="algo_select">
       <item>
        <property name="text">
         <string>Scan Line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Flood Fill</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Boundary Fill</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Grid Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="grid_size">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the size of the grid cells&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clear">
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_fill">
       <property name="text">
        <string>Clear Fill</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>751</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="mouse_movement">
       <property name="text">
        <string>Mouse Position:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mouse_pressed">
       <property name="text">
        <string>Last Click:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="time_algo">
       <property name="text">
        <string>Algorithm Time: N/A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>751</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="draw_grid">
       <property name="text">
        <string>Draw Grid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_polygon">
       <property name="text">
        <string>Start Polygon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_polygon">
       <property name="text">
        <string>Close Polygon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fill_polygon">
       <property name="text">
        <string>Fill Polygon</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>my_label</class>
   <extends>QLabel</extends>
   <header>my_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>